jscomp/main/jsoo_common.ml

```diff
@@ -32,20 +32,44 @@ module Js = struct
   external to_bytestring : js_string t -> string = "caml_js_to_byte_string"
 end
 
-let mk_js_error (loc : Location.t) (msg : string) =
+let string_of_formatted ~f x =
```

Comment:
is this `Format.asprintf`?

Reply:
https://github.com/melange-re/melange/pull/538/commits/9329c8714320bcf9988f1792c243d7b5af89be8f